src/xrGame/DiscordRPC.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <DiscordRPC/discord_register.h>
+#include <DiscordRPC/discord_rpc.h>
+#include <Windows.h>
+
+class DiscordRPC
+{
+public:
+    DiscordRPC() = default;
+    ~DiscordRPC();
+
+    void Init();
+    void Update(const char* level_name);
```

Comment:
We use `pcstr` alias for `const char*`.
```suggestion
    void Update(pcstr level_name);
```